aospy/calc.py

```diff
@@ -199,16 +199,11 @@ def __init__(self, proj=None, model=None, run=None, ens_mem=None, var=None,
         else:
             self.start_date = utils.times.ensure_datetime(date_range[0])
             self.end_date = utils.times.ensure_datetime(date_range[-1])
-        self.date_range = utils.times.create_monthly_time_array(
-            self.start_date, self.end_date, self.intvl_out
-        )
-        # Workaround for limited date range support due to nanosecond
-        # precison.  See https://github.com/spencerahill/aospy/issues/98
-        self.start_date_xarray = utils.times.numpy_datetime_range_workaround(
-            self.start_date)
-        self.end_date_xarray = (self.start_date_xarray +
-                                (self.end_date - self.start_date))
 
+        # What was this used for (it isn't used anymore)?
```

Comment:
@spencerahill I've commented out this code here; it appears that ```self.date_range``` is never used by ```Calc```.   In addition, this means that we never use ```utils.times.create_monthly_time_array```.  Should we delete this code?